src/common/acl_util.c

```diff
@@ -0,0 +1,331 @@
+/*
+ * (C) Copyright 2019 Intel Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+ * The Government's rights to use, modify, reproduce, release, perform, display,
+ * or disclose this software are subject to the terms of the Apache License as
+ * provided in Contract No. 8F-30005.
+ * Any reproduction of computer software, computer software documentation, or
+ * portions thereof marked with this legend must also reproduce the markings.
+ */
+
+/**
+ * Utility functions that may be used when interacting with Access Control
+ * Lists
+ */
+
+#include <daos_security.h>
+#include <gurt/common.h>
+#include <gurt/debug.h>
+#include <pwd.h>
+#include <grp.h>
+
+static size_t DEFAULT_BUF_LEN = 1024;
```

Comment:
Seemed more straightforward to maintain the privacy of the variable that way and avoid the potential for name conflicts, since the name is generic. I can change it to a macro if you prefer.

Reply:
No need to change it, but something like "#define DEFAULT_BUF_LEN 1024" would just be private to this file, now?

Reply:
Ah, for some reason I was thinking macros bled out of source files. Maybe on other compilers I've used.

Reply:
No, only if they are in header files or otherwise included in the code.   They have no visibility outside of a single compilation unit.